billing-service/src/report_helpers.py

```
@@ -0,0 +1,31 @@
+
+
+def tax(amount):
+    return amount * 0.18
+
+
+def late_fee(amount):
+    return amount * 0.18
+
+
+def is_overdue(days):
+    if days > 30 == True:
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)